### PR TITLE
lodestar beacon

### PR DESCRIPTION
**Pull request by gbayasgalan**

### CODE REVIEW

File: compose-examples/lodestar/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+* Get to know [Lodestar](https://chainsafe.github.io/lodestar/) a bit
+* Server with 4 (v)cpus & 8 gb memory & 100 gb storage
+
+## Services
+* geth (beacon connects to it to see deposits for validators)
+* beacon
+
+**All services are enabled by default.**
+
+## Validator accounts with launchpad
```
Comment (stefa2k) [inline compose-examples/lodestar/README.md:13]: Please remove until staking with loadstar is working

**Review by stefa2k — CHANGES_REQUESTED** [review]

File: compose-examples/lodestar/README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
+# Prysm standalone fullstack
```
Comment (stefa2k) [inline compose-examples/lodestar/README.md:1]: Prysm is wrong here

**Review by stefa2k — CHANGES_REQUESTED** [review]

File: compose-examples/lodestar/README.md (hunk 1 above)
Comment (gbayasgalan) [reply compose-examples/lodestar/README.md]: okey, got it and just removed

File: compose-examples/lodestar/README.md (hunk 2 above)
Comment (gbayasgalan) [reply compose-examples/lodestar/README.md]: my bad, now it's Lodestar:)

**Review by stefa2k — APPROVED** [review]
lgtm

**Pull request merged**